reference.rst

```diff
@@ -11,6 +11,7 @@ Reference
    reference/commands
    reference/conanfile
    reference/conanfile_txt
+   reference/conanws
```

Comment:
Maybe this can be called ``reference/workspace`` too? Or ``reference/workspace_files``? It is possible that the ``conanfile.py`` will be extracted, from ``conanws.py``, and it should be documented here most likely.